apps/vanilla/src/pages/2/logick/resultDisplay.ts

```diff
@@ -0,0 +1,97 @@
+import { PlayerInterface, Subscriber } from './types';
+
+/**
+ * Result Display creates layout and updates it.
+ * Result Display is subscriber of Player from './player.ts'.
+ */
+export class ResultDisplay implements Subscriber<PlayerInterface> {
+	/** Link to tag with id='game'. */
+	private app = document.getElementById('game');
```

Comment:
```suggestion
	private readonly app = document.getElementById('game');
```
Please, revise all your property to add `readonly` where it is possible.